app/assets/config/manifest.js

```diff
@@ -0,0 +1,3 @@
+//= link_tree ../images
```

Comment:
Now downgrade to Rails 5.0.2 and ruby version is also not binding to 2.4, I do the pay testing and all works as expected.

For manifest.js, it still can be exist even in 5.0.2, so I still prefer keep it to make further upgrade easier.